src/pylero/work_item.py

```diff
@@ -636,7 +636,9 @@ def add_hyperlink(self, url, role):
             self.uri, url, suds_role
         )
 
-    def add_linked_item(self, linked_work_item_id, role, revision=None, suspect=None):
+    def add_linked_item(
+        self, linked_work_item_id, role, revision=None, suspect=None, project_id=None
```

Comment:
Could you update the docstring with add the new project_id parameter also.

Reply:
oops. Sure will do!